app/helpers/layouts_helper.rb

```diff
@@ -0,0 +1,5 @@
+module LayoutsHelper
+  def prison_switcher_name_format(prison)
+    prison.name.split[1] + ' (HMP)'
```

Comment:
If we do need to keep this helper (i.e. we don't just change the prison names in the database), it looks like there's a bug with the current implementation.

`prison.name.split[1]` splits the name into words, and uses the second word.

This works for prison names which only have two words, e.g. `HMP Leeds`. This is correctly formatted as `Leeds (HMP)`.

However some prison names consist of more than one word, e.g. `HMP Isle of Wight`. This is incorrectly formatted as `Isle (HMP)` and you lose the `of Wight` bit.

If we keep this helper: it'd be good to have a test case to cover this.

Reply:
Also worth noting that this formatting might be slightly odd for `HMP Peterborough (Female)`.

It'll become `Peterborough (Female) (HMP)`.

Is that what DPS does, too?